core-java-modules/core-java-string-algorithms-3/README.md

```diff
@@ -5,3 +5,4 @@ This module contains articles about string-related algorithms.
 ### Relevant Articles:
 
 - [Check if Two Strings are Anagrams in Java](https://www.baeldung.com/java-strings-anagrams)
+- [Email Validation in Java](https://drafts.baeldung.com/wp-admin/post.php?post=101820&action=edit)
```

Comment:
Hi @gupta-ashu01 - let's please revert the README file. The editorial team will take care of that once the article is published.